s3torchconnectorclient/python/src/s3torchconnectorclient/_mountpoint_s3_client.pyi

```diff
@@ -16,6 +16,7 @@ class MountpointS3Client:
     def __init__(
         self,
         region: str,
+        endpoint: str,
         user_agent_prefix: str = "",
```

Comment:
A new `endpoint` read only attribute was added to this class, it should be added to the list of attributes at the top

Reply:
Added endpoint to class member variable